### DISCUSSION TITLE
Pivot Function in SlickGrid

### DISCUSSION THREAD

**Discussion by 6pac** [6pac/SlickGrid#585] [General]
Hey @ghiscoding , I seem to remember you said you had built a pivot function for slickgrid. I've checked a few of your repos and can't find any examples. Am I imagining it?

I'm looking at introducing a pivot function, specifically when editing data, to my application framework.
At the moment I'm trying to work out if I'm better off doing the pivot on the server, and then the de-pivot after editing - in that case SlickGrid wouldn't even need to know about it, or if that would be better done on the client side as part of SlickGrid.

**Comment by ghiscoding** [comment]
Hey there, are you talking about dynamically pinning a column? If so, you can test it in this [Example 12](https://ghiscoding.github.io/slickgrid-universal/#/example12), what I'm doing here is just changing the `frozenColumn` dynamically via `setOptions`.

![image](https://user-images.githubusercontent.com/643976/111054597-c4863a00-843b-11eb-8870-5e820fcd0a0a.png)

![image](https://user-images.githubusercontent.com/643976/111054616-eda6ca80-843b-11eb-9d1a-11bacbfaeb58.png)

**Reply by 6pac** [reply]
Actually, no, I mean a pivot, like this: https://www.lumeer.io/pivot-table-complete-guide/

**Reply by ghiscoding** [reply]
Unfortunately no, I never used pivot in any grids and I also looked at all the GerHobbelt grids and there's none in there either though I know ag-grid has that feature.

**Reply by 6pac** [reply]
No worries, thanks for letting me know. I suppose it's more useful if I build it as part of Slickgrid, then.
